e2e/suite1/specs/inspect-drawer.spec.ts

```diff
@@ -75,36 +75,36 @@ const expectDrawerTabsAndContent = () => {
         expect(li.text()).equals('Data');
       });
       e2e.components.PanelInspector.Data.content().should('be.visible');
-      e2e.components.PanelInspector.Stats.content().should('not.be.visible');
-      e2e.components.PanelInspector.Json.content().should('not.be.visible');
-      e2e.components.PanelInspector.Query.content().should('not.be.visible');
+      e2e.components.PanelInspector.Stats.content().should('not.exist');
```

Comment:
Is there any documentation from Cypress that mentions this change?

Reply:
I was in the middle of updating the PR after pushing my changes, just finished updating it 👍

Reply:
Awesome, thank you 🎉